soccer/planning/InterpolatedPath.hpp

```diff
@@ -59,10 +63,10 @@ namespace Planning
 			float length(const Geometry2d::Point &pt) const;
 
 			/** Returns number of waypoints */
-			size_t size() const { return points.size(); }
+			size_t size() const;
```

Comment:
Just FYI - for really small methods like this, if you keep it in the header, the compiler will automatically inline all calls to it.  Otherwise, the compiler can only inline calls to it that exist in the same .cpp file where the method is defined.